assets/src/edit-story/components/dropTargets/karma/background.karma.js

```diff
@@ -367,6 +363,121 @@ describe('Background Drop-Target integration', () => {
           expect(combinedBgTransform2).toBe('matrix(-1, 0, 0, 1, 0, 0)');
         });
       });
+
+      describe('when there is a third flipped image on the canvas', () => {
+        let flippedImageData;
+
+        beforeEach(async () => {
+          await fixture.events.click(fixture.editor.library.media.item(2));
+          flippedImageData = (await getElements(fixture))[2];
+
+          await fixture.events.click(
+            fixture.editor.inspector.designPanel.sizePosition.flipHorizontal
+          );
+
+          const element = fixture.editor.canvas.displayLayer.display(
+            flippedImageData.id
+          ).node;
+
+          await fixture.events.mouse.seq(({ moveRel, down, up }) => [
+            moveRel(element, '50%', '50%'),
+            down(),
+            moveRel(element, '50%', '200%'),
+            up(),
+          ]);
+        });
+
+        it('should correctly handle flipped image dropped into non-flipped image', async () => {
+          // Verify that non-flipped element has the correct transform (flip) before doing anything
+          let target = fixture.editor.canvas.displayLayer.display(imageData.id)
+            .node;
+          let rep = fixture.editor.canvas.displayLayer.display(imageData.id)
+            .replacement;
+          const targetImg = target.querySelector('img');
+          const transformBefore = getAllTransformsBetween(targetImg, target);
+          expect(transformBefore).toBe('');
+
+          // Drag the image element to the background
```

Comment:
Comment is for the background here, should be for the non-flipped image.

Reply:
Fixed in 61002303e.